lib/test/commands/test_comm_control.cpp

```diff
@@ -22,22 +22,25 @@ class TestCommControl : public ScrutinyTest
 	virtual void SetUp()
 	{
 		config.max_bitrate = 0x12345678;
+		config.set_display_name("hello");
 		scrutiny_handler.init(&config);
 	}
 };
 
 
 TEST_F(TestCommControl, TestDiscover)
 {
+
 	ASSERT_FALSE(scrutiny_handler.comm()->is_connected());   // We should get a Discover response even when not connected.
 	ASSERT_EQ(sizeof(scrutiny::protocol::CommControl::DISCOVER_MAGIC), 4u);
-	uint8_t request_data[8 + 4] = { 2,1,0,4 };
+	uint8_t request_data[8 + 4] = { 2,1,0,4};
 	std::memcpy(&request_data[4], scrutiny::protocol::CommControl::DISCOVER_MAGIC, sizeof(scrutiny::protocol::CommControl::DISCOVER_MAGIC));
 
 
 	uint8_t tx_buffer[64];
-	uint8_t expected_response[9 + 32] = { 0x82,1,0,0,32 };   // Version 1.0
+	uint8_t expected_response[9 + 32+5] = { 0x82,1,0,0,32+5};   // Version 1.0
```

Comment:
lots of magic numbers. Maybe add /* */ style comment next to each number to explain where they come from?
```suggestion
	uint8_t expected_response[
	  9  /* header */
	  +32 /* ?? */
	  +5 /* display name */
	] = { 0x82,1,0,0,32+5};   // Version 1.0
```

Reply:
also, they initialisation value is way short (5 bytes) compared to the array length (46)

Reply:
If comments are to be added, then the whole unit test foldet should be done. Will do eventually

For initialisation, unspecified bytes are 0 per c++ standard.

Reply:
Sounds good!